gulpfile.js

```diff
@@ -80,25 +80,33 @@ gulp.task('test.e2e', ['test.compile'], function(done) {
 
```

Comment:
Not necessarily now, but shouldn't we move this over to something that globs all *_test.ts (or *_e2e_test.ts or so), and considers all the other files to be imported libraries/support code for those tests? I don't think putting everything into one file will scale medium term.